src/SprykerSdk/Zed/ComposerReplace/Communication/Console/ComposerReplaceConsole.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\ComposerReplace\Communication\Console;
+
+use Generated\Shared\Transfer\ComposerPackageTransfer;
+use Generated\Shared\Transfer\ComposerReplaceResultCollectionTransfer;
+use Spryker\Zed\Kernel\Communication\Console\Console;
+use SprykerSdk\Zed\ComposerReplace\ComposerReplaceConfig;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \SprykerSdk\Zed\ComposerReplace\Business\ComposerReplaceFacadeInterface getFacade()
+ */
+class ComposerReplaceConsole extends Console
+{
+    public const COMMAND_NAME = 'code:composer:replace';
```

Comment:
Can we use

    dev:composer:replace

please.
We use dev for all our (new) own tools to avoid them being mixed up with normal code tools.